docs/organizing-and-exploring-results-in-the-ui/searching-experiments.rst

```diff
@@ -1,17 +1,31 @@
+Searching and filtering experiments
+===================================
+
+Basic Filtering
+---------------
+You can perform the simplest filtering by typing into the search fields:
+
+   .. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/search_fields.png
+      :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/search_fields.png
```

Comment:
A lot is going on on this screen-shot. I'm afraid that reader does not know where to look for this search.